CI/macos/build_ffmpeg.sh

```diff
@@ -72,8 +72,8 @@ _build_product() {
             --enable-cross-compile --enable-shared --disable-static --enable-pthreads \
             --shlibdir="${BUILD_DIR}/lib" --pkg-config-flags="--static" --prefix="${BUILD_DIR}" --enable-rpath \
             --host-cflags="-I${BUILD_DIR}/include" --host-ldflags="-L${BUILD_DIR}/lib"  \
-            --extra-ldflags="-target x86_64-apple-macos${DARWIN_TARGET} -L${BUILD_DIR}/lib -lstdc++" \
-            --extra-cflags="-fno-stack-check -target x86_64-apple-macos${DARWIN_TARGET} -I${BUILD_DIR}/include" \
+            --extra-ldflags="-target x86_64-apple-macos10.13 -L${BUILD_DIR}/lib -lstdc++" \
+            --extra-cflags="-fno-stack-check -target x86_64-apple-macos10.13 -I${BUILD_DIR}/include" \
```

Comment:
Could we use `MACOSX_DEPLOYMENT_TARGET` or some equivalent here?  Not sure about the arm64 branch.